### ISSUE TITLE
Is settings

### ISSUE THREAD

**Issue by Nedjii** [BentoBoxWorld/BSkyBlock#223]
Hello, when a player tries to manage ranks permissions in /is settings where you can tell which island rank can invite players etc it tells him that he doesn't have the permission to but I found no permission for that,
regards.

**Comment by tastybento** [comment]
Doesn't it show in chat?
<img width="588" alt="Screen Shot 2019-11-30 at 4 13 54 PM" src="https://user-images.githubusercontent.com/4407265/69907604-71928b80-138c-11ea-883f-3a5972905d8f.png">

**Comment by Nedjii** [comment]
No, I modified the locale, that's why, I thought it was an issue, my bad.

**Comment by Nedjii** [comment]
Wait, why is it noticed admin in the permission, isn't that supposed to be accessed by the island owner ?
